css/index.css

```diff
@@ -1 +1,393 @@
-/* Use your CSS from RWD - I here.  IF you don't have CSS, use the solution CSS from RWD - I*/
\ No newline at end of file
+/* http://meyerweb.com/eric/tools/css/reset/
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    font-size: 100%;
+    font: inherit;
+    vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure,
+footer, header, hgroup, menu, nav, section {
+    display: block;
+}
+body {
+    line-height: 1;
+}
+ol, ul {
+    list-style: none;
+}
+blockquote, q {
+    quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+    content: '';
+    content: none;
+}
+table {
+    border-collapse: collapse;
+    border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+html {
+    font-size: 62.5%;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+h2 {
+    font-size: 1.9rem;
+}
+
+p {
+    line-height: 1.25;
+    font-size: 1.6rem;
+}
+
+.main-container {
+    max-width: 880px;
+    margin-left: auto;
+    margin-right: auto;
+    height: 100%;
+}
+
+nav {
+    background-color: white;
+    height: 106px;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    z-index: 1;
+}
+
+.nav-item-wrapper {
+    display: inline-flex;
+    align-items: center;
+    justify-content: space-between;
+    height: 100%;
+    font-size: 1.6rem;
+    /*padding-left: 10px;*/
+    vertical-align: top;
+    width: 69%;
+}
+
+nav a {
+    color: gray;
+    text-decoration: none;
+}
+
+.logo-wrapper {
+    display: inline-flex;
+    align-items: center;
+    justify-content: flex-end;
+    height: 100%;
+    width: 30%;
+}
+
+header {
+    display: flex;
+    justify-content: space-around;
+    height: 455px;
+    border-bottom: 2px solid;
+    margin-bottom: 30px;
+    margin-top: 136px;
+}
+
+/*.header-container {*/
+/*!* TODO update to flexbox!*/
+/*}*/
+
+.header-wrapper {
+    position: relative;
```

Comment:
this is an odd choice. You doing a lot of extra work double wrapping this content and using absolute positioning and everything when all you have to do is remember your cross axis on the header flex and you are good to go.